kobo/rpmlib.py

```diff
@@ -69,7 +71,7 @@ def get_rpm_header(file_name, ts=None):
             ts.setKeyring(rpm.keyring())
         ts.setVSFlags(rpm._RPMVSF_NOSIGNATURES|rpm._RPMVSF_NODIGESTS)
 
-    if type(file_name) in (str, unicode):
+    if type(file_name) in (str, six.text_type):
```

Comment:
Should this maybe be changed to `isinstance(file_name, six.string_types)`?

Reply:
Yes, you are right. I fixed this in the latest fixup commit. Thanks!